frontend/src/components/ShootMessages/K8sExpirationMessage.vue

```diff
@@ -7,12 +7,9 @@ SPDX-License-Identifier: Apache-2.0
 <template>
   <div>
     <span v-if="isValidTerminationDate">Kubernetes version of this cluster expires
-      <v-tooltip right>
-        <template v-slot:activator="{ on }">
-          <time-string v-on="on" :date-time="expirationDate" mode="future" class="font-weight-bold"></time-string>
-        </template>
-        {{getDateFormatted(expirationDate)}}
-      </v-tooltip>. </span>
+       <time-string :date-time="expirationDate" mode="future" date-tooltip content-class="font-weight-bold"></time-string>
```

Comment:
The tooltip was positioned right now it is top. I think we should allow to define the position on the time-string component. And it maybe would make sense to allow time strings without tooltip.

Reply:
I can add this to the component but right now I don't think we need it. We had it for the expiration messages but I don't see a reason to have it on the right side for this components. I think it is better to have it in a consistent position. If you have a good argument to position it other than top, I will add the option